fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -57,12 +57,6 @@ def generate_ios_command(params)
         end
 
         def generate_android_command(params)
-          # We have to generate an empty XML file to make the crashlytics CLI happy :)
-          require 'tempfile'
-          xml = Tempfile.new('xml')
-          xml.write('<?xml version="1.0" encoding="utf-8"?><manifest></manifest>')
-          xml.close
-
```

Comment:
@chuganzy Can we find a way to keep this inside of `generate_android_command`? It feels disconnect to pull this code out of this method. It would be great if there was a way for us to not have to do that 😇

Reply:
Hmm, there are many ways to solve that.
1: Make `generate_xxx_command` to return hash or some class that has `command` and `tempfiles`.
2: Make `generate_xxx_command` to receive block so that these functions can determine the scope.
3: Add new `android_manifest_path` parameter to this action and generate empty manifest in action as we do for the release notes if not specified.

No.3 might be the best. What do you think?